### PR TITLE
[#164630993] Hide sensitive info on iOS

### PR DESCRIPTION
**Pull request by fpersico**

### CODE REVIEW

File: ios/ItaliaApp/AppDelegate.m [hunk 1]
```diff
@@ -81,4 +81,34 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
   [RCTPushNotificationManager didReceiveLocalNotification:notification];
 }
 
+- (void)applicationWillResignActive:(UIApplication *)application {
+
+     // Fill screen with our own colour
+    UIView *colourView = [[UIView alloc]initWithFrame:self.window.frame];
+    colourView.backgroundColor = [UIColor whiteColor];
+    colourView.tag = 1234;
```
Comment (cloudify) [inline ios/ItaliaApp/AppDelegate.m]: is this number arbitrary? can we add some comment describing what's going on here? perhaps some reference too?

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#164630993](https://www.pivotaltracker.com/story/show/164630993): Disable background screenshot with sensitive information

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ios/ItaliaApp/AppDelegate.m (hunk 1 above)
Comment (fpersico) [reply ios/ItaliaApp/AppDelegate.m]: Yes it is arbitrary and is used here https://github.com/teamdigitale/io-app/pull/943/files#diff-8d59bb2633e5b2fda2c2d6853cc45662R103 to get a reference to the view.

Comment (cloudify) [reply ios/ItaliaApp/AppDelegate.m]: yes pls describe this mechanism with a short comment

Comment (fpersico) [reply ios/ItaliaApp/AppDelegate.m]: Done.

**Pull request merged**